main.go

```diff
@@ -41,6 +41,8 @@ var defaultStopTimeout = flag.Duration("default_stop_timeout", defaultSystemdSto
 var build = flag.Bool("build", false, "if set, generated container build systemd services will be enabled.")
 var writeNixSetup = flag.Bool("write_nix_setup", true, "if true, Nix setup code is written to output (runtime, DNS, autoprune, etc.)")
 var autoFormat = flag.Bool("auto_format", false, `if true, Nix output will be formatted using "nixfmt" (must be present in $PATH).`)
+var optionPrefix = flag.String("option_prefix", "", "Prefix for the option. If empty, the project name will be used as the option name. (e.g. custom.containers)")
+var enableOption = flag.Bool("enable_option", false, "Generate a nix option, which allows enabling the components elements of the project.")
```

Comment:
Change help text to:

generate a NixOS module option. this allows you to enable or disable the generated module from within your NixOS config. by default, the option will be named "options.[project_name]", but you can add a prefix using the "option_prefix" flag.